### ISSUE TITLE
Missing files are erroneously reported as a PASS

### ISSUE THREAD

**Issue by jrwagz** [igorshubovych/markdownlint-cli#370]
As you'd expect, if a file exists with errors, we exit with `1` to report the error, however if the file provided does not exist, then the help menu is printed, and we exit with a code of `0`.  When this happens, we should instead exit with a code of `1` to indicate the error.

Bad behavior:
```
~/projects/markdownlint-cli$ docker run -v ${PWD}:${PWD} --workdir ${PWD} markdownlint-local:latest not-a-real-file.md
Usage: markdownlint [options] <files|directories|globs>

MarkdownLint Command Line Interface

Options:
  -V, --version                               output the version number
  -c, --config [configFile]                   configuration file (JSON, JSONC, JS, or YAML)
  -d, --dot                                   include files/folders with a dot (for example `.github`)
  -f, --fix                                   fix basic errors (does not work with STDIN)
  -i, --ignore [file|directory|glob]          file(s) to ignore/exclude (default: [])
  -j, --json                                  write issues in json format
  -o, --output [outputFile]                   write issues to file (no console)
  -p, --ignore-path [file]                    path to file with ignore pattern(s)
  -q, --quiet                                 do not write issues to STDOUT
  -r, --rules  [file|directory|glob|package]  include custom rule files (default: [])
  -s, --stdin                                 read from STDIN (does not work with files)
  --enable [rules...]                         Enable certain rules, e.g. --enable MD013 MD041 --
  --disable [rules...]                        Disable certain rules, e.g. --disable MD013 MD041 --
  -h, --help                                  display help for command
~/projects/markdownlint-cli$ echo $?
0
```

Expected behavior:
```
~/projects/markdownlint-cli$ docker run -v ${PWD}:${PWD} --workdir ${PWD} markdownlint-local:latest not-a-real-file.md
Usage: markdownlint [options] <files|directories|globs>

MarkdownLint Command Line Interface

Options:
  -V, --version                               output the version number
  -c, --config [configFile]                   configuration file (JSON, JSONC, JS, or YAML)
  -d, --dot                                   include files/folders with a dot (for example `.github`)
  -f, --fix                                   fix basic errors (does not work with STDIN)
  -i, --ignore [file|directory|glob]          file(s) to ignore/exclude (default: [])
  -j, --json                                  write issues in json format
  -o, --output [outputFile]                   write issues to file (no console)
  -p, --ignore-path [file]                    path to file with ignore pattern(s)
  -q, --quiet                                 do not write issues to STDOUT
  -r, --rules  [file|directory|glob|package]  include custom rule files (default: [])
  -s, --stdin                                 read from STDIN (does not work with files)
  --enable [rules...]                         Enable certain rules, e.g. --enable MD013 MD041 --
  --disable [rules...]                        Disable certain rules, e.g. --disable MD013 MD041 --
  -h, --help                                  display help for command
~/projects/markdownlint-cli$ echo $?
1
```

**Comment by DavidAnson** [comment]
This is a duplicate of the issue you identify. Passing an explicit file name is basically the same as a glob that matches a single item.
